app/views/todos/_form.html.erb

```diff
@@ -21,6 +21,15 @@
     <% form.text_field current_user.id %>
   </div>
 
+  <% if @todo.estate != nil and @todo.estate!= true %>
```

Comment:
This line can be refactored to `<% unless @todo.estate %>`

Reply:
change it, thanks